salt/modules/selinux.py

```diff
@@ -616,7 +616,7 @@ def _fcontext_add_or_delete_policy(
     if "add" == action:
         # need to use --modify if context for name file exists, otherwise ValueError
         filespec = re.escape(name)
-        cmd = f"semanage fcontext -l | egrep '{filespec}'"
+        cmd = f"semanage fcontext -l | egrep '{filespec} '"
```

Comment:
wondering about the extra space added just before the single-quote and double-quote at the end ?

Reply:
That is the point of this change.
If there is already a fcontext registered for `abcd` and you attempt to add one for `abc` the lack of that extra space incorrectly causes the existing fcontext to be matched by the grep - incorrectly reporting that the fcontext is already registered.

Reply:
Remember vaguely something along those lines from last year when working on semanage in selinux environment